chia/wallet/vc_wallet/vc_wallet.py

```diff
@@ -103,7 +104,12 @@ async def coin_added(self, coin: Coin, height: uint32, peer: WSChiaConnection) -
                 f"Cannot get verified credential coin: {coin.name().hex()} puzzle and solution"
             )  # pragma: no cover
             return  # pragma: no cover
-        vc = VerifiedCredential.get_next_from_coin_spend(cs)
+        try:
+            vc = VerifiedCredential.get_next_from_coin_spend(cs)
+        except Exception as e:  # pragma: no cover
+            self.log.debug(
+                f"Syncing VC from coin spend failed (likely means it was revoked): {e}\n{traceback.format_exc()}"
+            )
```

Comment:
shouldn't this return? otherwise it will run into and error in the next statement

Reply:
oops yeah

Reply:
fixed